Program.cs

```diff
@@ -5,11 +5,91 @@ namespace Calculator
   public static class Program
   {
     private static void Main(string[] args)
+    {
+      PrintWelcomeMessage();
+
+      while (true)
```

Comment:
Would be easier for the user if it was possible to exit from the program by entering a specific character. Currently, this will require the user to shut down the console window to quit.

Reply:
Thank you. It is really effective.